third_party/ruby_root.bzl

```diff
@@ -0,0 +1,20 @@
+# This rule is used to get the RUBY_ROOT environment variable from the
+# user's local environment and make it available to the build system.
+def _ruby_root_impl(repository_ctx):
+  # Retrieve the RUBY_ROOT environment variable
+  ruby_root = repository_ctx.os.environ.get("RUBY_ROOT")
+
+  if not ruby_root:
```

Comment:
Are you sure this can ever fail?

`dict.get(missing_key)` throws in Python, but IDK about the specific type of `repository_ctx.os.environ`

Reply:
Yes I actually saw it fail locally when I misspelled "RUBY_ROOT" above 😆